lib/Token/Token.cpp

```diff
@@ -232,16 +244,26 @@ std::string Token::getHash() const {
   hash += this->getName().str();
   hash += this->getFunctionName();
   hash += this->getMemTypeName();
-  hash += this->getFieldIndex();
+//  hash += this->getFieldIndex();   
+  return hash;
+}
+
+std::string Token::getHashNoFieldIdx() const {
```

Comment:
I do not think we should calculate a separate hash and if we are doing then it should have some internal use?

Reply:
I have removed this function.